src/index.html

```diff
@@ -11,7 +11,219 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      onsubmit="onSubmit()"
+    >
+
+      <fieldset class="box">
+        <legend>Personal information</legend>
+
+        <div class="labels">
+          <label for="surname">
+            Surname:
+            <input
+              id="surname"
+              type="text"
+              name="surname"
+              autocomplete="off"
+              required
+              minlength="1"
+            >
+          </label>
+        </div>
+
+        <div class="labels">
+          <label for="name">
+            Name:
+            <input
+              type="text"
+              name="name"
+              autocomplete="off"
+              id="name"
+              required
+              minlength="1"
+            >
+          </label>
+        </div>
+
+
+        <div class="labels">
+          <label for="age">
+            How old are You?
```

Comment:
one blank line maximum

```suggestion
        </div>

        <div class="labels">
          <label for="age">
            How old are You?
```